pkg/dl/dl.go

```diff
@@ -24,30 +24,43 @@ import (
 // #include <stdlib.h>
 import "C"
 
+// RTLD constants
 const (
-	RTLD_LAZY = C.RTLD_LAZY
-	RTLD_NOW = C.RTLD_NOW
-	RTLD_GLOBAL = C.RTLD_GLOBAL
-	RTLD_LOCAL = C.RTLD_LOCAL
+	// Deprecated, don't use ALL_CAPS in Go names
+	RTLD_LAZY     = C.RTLD_LAZY
```

Comment:
If we change the following line in `gen/init.go`:
```
	nvmlLibraryLoadFlags = dl.RTLD_LAZY | dl.RTLD_GLOBAL
```
We can remove the deprecated variable names here.

Or alternatively, what about rather adding a constant:
```
DefaultRtldFlags = C.RTLD_LAZY | C.RTLD_GLOBAL
```
here and using this instead?

Reply:
No sure whether you care about backward compatibility or not, so I just keep as it is.

Reply:
Yes, you are right. The golang `debug/elf` package also contains such constants.

https://github.com/golang/go/blob/9de1ac6ac2cad3871760d0aa288f5ca713afd0a6/src/debug/elf/elf.go#L54-L62